process-compose.yaml

```diff
@@ -662,3 +662,12 @@ processes:
     depends_on:
       sequencer1:
         condition: process_healthy
+
+  dev-rollup:
+    command:
+      dev-rollup register --ns 1; dev-rollup register --ns 2; dev-rollup register --ns 3
```

Comment:
There might be a better name for the binary than dev-rollup. I'm also considering something like dev-rollup register --ns 1 --ns 2 --ns 3, but I'm not sure if there's a clean way to implement it and if that would be better, as each register also has other optional parameters. I might keep it as it is for now.